src/app/routes/registration/request-claim/request-claim.component.ts

```diff
@@ -178,83 +215,59 @@ export class RequestClaimComponent implements OnInit, SubjectElements {
   }
 
   async submit() {
+    if (!this.enrolmentForm.valid && !this.isRegistrationTypeSelected()) {
```

Comment:
Oh, is it because there are two separate forms for some reason?

Reply:
Ok, i think i can merge them. I did two separate forms because i thought that onchain/offchain is only for `enrol for myself`. However yesterday you said that it can also be for assets. So i will adjust this also.

Reply:
@DawidGil-jlabs If not able to merge the forms, I think it's okay, perhaps just add a comment to explain
```suggestion
    // RegistrationType check is separate from enrolmentForm.valid as merging forms caused updating issues
    if (!this.enrolmentForm.valid && !this.isRegistrationTypeSelected()) {
```